test/api/syncmer_hash_test.cpp

```diff
@@ -53,31 +71,40 @@ using underlying_range_types = ::testing::Types<std::vector<seqan3::dna4>,
                                                 std::list<seqan3::dna4>,
                                                 std::list<seqan3::dna4> const>;
 
-TYPED_TEST_SUITE(syncmer_hash_view_properties_test, underlying_range_types, );
+TYPED_TEST_SUITE(open_view_properties_test, underlying_range_types, );
 
 class syncmer_hash_test : public ::testing::Test
 {
 protected:
     std::vector<seqan3::dna4> text1{"AAAAAA"_dna4};
-    result_t result1{0, 0};
+    result_t result1_open{0, 0};
 
     std::vector<seqan3::dna4> text3{"ACGGCGACGTTTAG"_dna4};
-//         Kmers(2,5):    ACGGC CGGCG GGCGA GCGAC CGACG GACGT ACGTT CGTTT GTTTA TTTAG
-//        Hashed(2,5):    105,  422,  664,  609,  390,  539,  111,  447,  764,  1010
-//      Syncmers(2,5):    ACGGC CGGCG       GCGAC             ACGTT CGTTT GTTTA TTTAG
-//  Syncmer stop(2,5):    ACGGC CGGCG       GCGAC
-// Syncmer start(2,5):                                        ACGTT CGTTT GTTTA TTTAG
-    result_t result3_ungapped{105,422,609,111,447,764,1010};
-    result_t result3_ungapped_stop{105,422,609};
-    result_t result3_ungapped_start{111,447,764,1010};
-//         Kmers(1,4):    ACGG  CGGC  GGCG  GCGA  CGAC  GACG  ACGT  CGTT  GTTT  TTTA  TTAG
-//        Hashed(1,4):    26,   105,  166,  152,  97,   134,  27,   111,  191,  252,  242
-//      Syncmers(1,4):    ACGG  CGGC        GCGA              ACGT  CGTT  GTTT  TTTA
-//  Syncmer stop(1,4):    ACGG  CGGC        GCGA
-// Syncmer start(1,4):                                        ACGT  CGTT  GTTT  TTTA
-    result_t result3_14{26,105,152,27,111,191,252};
-    result_t result3_14_stop{26,105,152};
-    result_t result3_14_start{27,111,191,252};
+//             Kmers(2,5):    ACGGC CGGCG GGCGA GCGAC CGACG GACGT ACGTT CGTTT GTTTA TTTAG
+//            Hashed(2,5):    105,  422,  664,  609,  390,  539,  111,  447,  764,  1010
+//          Syncmers(2,5):    ACGGC CGGCG                         ACGTT CGTTT GTTTA
+//      syncmer stop(2,5):    ACGGC CGGCG
+//     syncmer start(2,5):                                        ACGTT CGTTT GTTTA
+//        c-Syncmers(2,5):    ACGGC CGGCG       GCGAC             ACGTT CGTTT GTTTA TTTAG
+//    c-syncmer stop(2,5):    ACGGC CGGCG       GCGAC
+//   c-syncmer start(2,5):                                        ACGTT CGTTT GTTTA TTTAG
+    result_t result3_open{105,422,111,447,764};
+    result_t result3_stop_open{105,422};
+    result_t result3_start_open{111,447,764};
+    result_t result3_closed{105,422,609,111,447,764,1010};
+    result_t result3_stop_closed{105,422,609};
+    result_t result3_start_closed{111,447,764,1010};
+//             Kmers(1,4):    ACGG  CGGC  GGCG  GCGA  CGAC  GACG  ACGT  CGTT  GTTT  TTTA  TTAG
+//            Hashed(1,4):    26,   105,  166,  152,  97,   134,  27,   111,  191,  252,  242
+//          Syncmers(1,4):    ACGG  CGGC  cgcc              cgtc  ACGT  aagc  aaac  aaat
```

Comment:
I'm not sure I understand these comments here. Are we considering the reverse strand?